pkgs/development/python-modules/vinput/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  setuptools,
+  wheel,
+  testers,
+  vinput,
+}:
+
+buildPythonPackage rec {
+  pname = "vinput";
+  version = "0.6.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "xslendix";
+    repo = "libvinput.py";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-86oNsbdw59zvxQvn05Q493mAoZdIBWUu/GtQvIJiEEY=";
+  };
+
+  postPatch = ''
+    substituteInPlace "vinput/_binding.py" \
+      --replace-fail  "dir_path = os.path.dirname(os.path.realpath(__file__))" "dir_path = "$out/""
+  '';
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  pythonImports = [
```

Comment:
You mean `pythonImportsCheck`?

Reply:
Yes, it now fails cause it can't fetch the dll even from the hardcoded path, might have to mark this as a draft pr till I fully fix that issue